app/src/main/java/com/example/iphonesearchapi/viewmodel/ItunesViewModel.kt

```diff
@@ -17,14 +17,14 @@ class ItunesViewModel(private val serviceUtil: IphoneApiService) : ViewModel() {
     fun triggerItunesapi(searchText: String) {
 
         viewModelScope.launch {
-            _itunes.setValue (ResultOf.Loading)
+            _itunes.value = ResultOf.Loading
```

Comment:
As a general practice try to avoid using `.value` on live data when you are inside of a coroutine context.